package.json

```diff
@@ -4,12 +4,14 @@
   "description": "Sample App for JavaScript SDK view layer for Onfido identity verification",
   "scripts": {
     "predev": "npm run build",
-    "dev": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https",
+    "dev:webpack": "NODE_ENV=development webpack-dev-server --inline --hot --progress --https --cert cert.pem --key key.pem",
```

Comment:
If the task is called directly and there are no `cert` and `key` present, then webpack self signs its server. But the task is supposed to be called by the node server, who generates the `cert` and `key`, uses it itself and calls webpack to use it too.

It's important for both the node server and the webpack server to share the certificate, because otherwise when you open the webapp, you accept the "dangerous" self certificate of the webpack server, but that does not mean you accept the node server self certificate as well. For that, you would have to open the node server in another page, making for a very poor UX.